site/layouts/partials/sidebar/submenu-begin.html

```diff
@@ -1,5 +1,5 @@
 
-<a href="{{ .href }}" class="list-group-item collapsed" data-toggle="collapse" data-parent="#sidebar" aria-expanded="false" style="z-index: inherit">
+<a href="{{ .href }}" class="list-group-item collapsed" data-toggle="collapse" data-parent="{{ .parent }}" aria-expanded="false" style="z-index: inherit">
```

Comment:
Looks like this can be fixed by updating the select function, line 75 in src/js/app.js